app/helpers/application_helper.rb

```diff
@@ -8,4 +8,12 @@ def date_to_hours(date)
   def format_date(date)
     date.strftime('%d/%m/%Y %H:%M')
   end
+
+  def manager?
+    current_user.role == 'manager'
```

Comment:
If you declared role as an enum then you do not have to write this helpers.
Simply `current_user.manager?` should do the job.
You can check more methods which comes with enums in [here](https://api.rubyonrails.org/v7.0.3.1/classes/ActiveRecord/Enum.html).